src/Microsoft.AspNet.Localization/RequestLocalizationOptions.cs

```diff
@@ -51,6 +51,33 @@ public RequestCulture DefaultRequestCulture
         }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to set a request culture to an ancestor culture in the case the
+        /// culture determined by the configured <see cref="IRequestCultureProvider"/>s is not in the
+        /// <see cref="SupportedCultures"/> list but an ancestor culture is.
+        /// Defaults to <c>true</c>;
+        /// </summary>
+        /// <example>
+        /// If this property is <c>true</c> and the application is configured to support the culture "fr", but not the
+        /// culture "fr-FR", and a configured <see cref="IRequestCultureProvider"/> determines a request's culture is
+        /// "fr-FR", then the request's culture will be set to the culture "fr", as it is an ancestor of "fr-FR".
+        /// </example>
+        public bool FallbackToAncestorCulture { get; set; } = true;
```

Comment:
@DamianEdwards ancestor and parent words might have these semantics, but I have to agree with @ryanbrandenburg that there is no mention about ancestor cultures anywhere (and it also has a temporal connotation), while parent cultures are well known concept to both .NET and native developers